bin/rose-check-software

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
I haven't looked at this, but I'm almost certain that this will need some work.

Reply:
https://github.com/wxtim/rose/pull/5

Reply:
I merged this, then lost it due to a foolish force push. I've added it again, referencing you in the commit msg.  :disappointed: